Sources/RInAppMessaging/Views/Components/ActionButton.swift

```diff
@@ -8,7 +13,8 @@ internal class ActionButton: UIButton {
         static let minFontSize: CGFloat = 10
         static let labelMargin = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
         static let cornerRadius: CGFloat = 8
-        static let borderWidth: CGFloat = 0
+        static let borderWidth: CGFloat = 0.5
+        static let borderColor = UIColor(hexString: "#D1D1D1")!
```

Comment:
Is this colour okay if the background colours are dark? What if the modal bg is also grey?